data/json/itemgroups/clothing.json

```diff
@@ -188,5 +188,85 @@
       [ "pocketwatch", 25 ],
       [ "wristwatch", 100 ]
     ]
+  },
+  {
+    "id": "clothing_work_boots",
+    "type": "item_group",
+    "subtype": "distribution",
+    "entries": [
+      { "item": "boots", "prob": 60 },
+      { "item": "boots_hiking", "prob": 10 },
+      { "item": "boots_steel", "prob": 30 }
+    ]
+  },
+  {
+    "id": "clothing_work_glasses",
+    "type": "item_group",
+    "subtype": "distribution",
+    "entries": [
+      { "group": "clothing_glasses", "prob": 10 },
+      { "item": "glasses_safety", "prob": 75 },
+      { "item": "goggles_welding", "prob": 15 }
+    ]
+  },
+  {
+    "id": "clothing_work_gloves",
+    "type": "item_group",
+    "subtype": "distribution",
+    "entries": [
+      { "item": "gloves_leather", "prob": 30 },
+      { "item": "gloves_rubber", "prob": 10 },
+      { "item": "gloves_work", "prob": 60 }
+    ]
+  },
+  {
+    "id": "clothing_work_hat",
+    "type": "item_group",
+    "subtype": "distribution",
+    "entries": [
+      { "item": "hat_hard", "prob": 60 },
+      { "item": "hat_hard_hooded", "prob": 15 },
+      { "item": "miner_hat", "prob": 10 },
+      { "item": "wearable_light", "prob": 15 }
+    ]
+  },
+  {
+    "id": "clothing_work_mask",
+    "type": "item_group",
+    "subtype": "distribution",
+    "entries": [
+      { "item": "mask_dust", "prob": 90 },
+      { "item": "mask_filter", "prob": 10 }
+    ]
+  },
+  {
+    "id": "clothing_work_set",
+    "type": "item_group",
+    "subtype": "collection",
+    "//": "Equipment used for heavy labour",
+    "items": [
+      { "group": "clothing_work_boots" },
+      { "group": "clothing_work_glasses", "prob": 60 },
+      { "group": "clothing_work_gloves", "prob": 75 },
+      { "group": "clothing_work_hat", "prob": 60 },
+      { "group": "clothing_work_mask", "prob": 40 },
+      { "item": "ear_plugs", "prob": 15 },
+      { "item": "tool_belt", "prob": 20 },
+      { "distribution": [
+        { "collection": [
+          { "item": "technician_shirt_blue" },
```

Comment:
I'm not so sure it matters that much. Perhaps that's was the company uniform for that technician? Keeping the itemgroups consistent is more important so that when somebody provides `clothing_indoor_set` they also have the same groups etc